File: food_delivery/src/main/java/com/datn/food_delivery/models/Review.java
package com.datn.food_delivery.models;

public class Review {
    private Long review_id;
    private int rating;
    private String email;
    private Long product_id;

    public Review() {
    }

    public Review(Long review_id, int rating, String email, Long product_id) {
        this.review_id = review_id;
        this.rating = rating;
        this.email = email;
        this.product_id = product_id;
    }

    public Long getReview_id() {
        return review_id;
    }

    public void setReview_id(Long review_id) {
        this.review_id = review_id;
    }

    public int getRating() {
        return rating;
    }

    public void setRating(int rating) {
        this.rating = rating;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public Long getProduct_id() {
        return product_id;
    }

    public void setProduct_id(Long product_id) {
        this.product_id = product_id;
    }
}

File: food_delivery_flutter/lib/Model/Review.dart
class Review{
  final int review_id;
  final int rating;
  final String? email;
  final int product_id;

  Review({required this.review_id, required this.rating, required this.email, required this.product_id});

  factory Review.fromJson(Map<String, dynamic> json) {
    return Review(
        review_id: json['review_id'],
        rating: json['rating'],
        email: json['email'],
        product_id: json['product_id'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'review_id': review_id,
      'rating': rating,
      'email': email,
      'product_id': product_id,
    };
  }

  factory Review.toJson(Map<String, dynamic> json) {
    return Review(
        review_id: json['review_id'],
        rating: json['rating'],
        email: json['email'],
        product_id: json['product_id'],
    );
  }
}